### ISSUE TITLE
antew/vim-elm-language-server merged into w0rp/ale

### ISSUE THREAD

**Issue by andys8** [Libbum/Brahman.files#8]
Once https://github.com/w0rp/ale/issues/2592 is merged and the plugin updated, `antew/vim-elm-language-server` can be replaced. The linter name is `elm_ls`. See https://github.com/elm-tooling/elm-language-server/pull/80/files for changes.

**Comment by Libbum** [comment]
Thanks for this, I'll keep an eye on it!
